File: codefights.com/encode.js
encode = s => {
    o='';
    l=s[0];
    for(c=i=1;i<=s.length;i++) {
        if(s[i]==l)
            c++;
        else {
            o+=l+c;
            l=s[i];
            c=1;
        }
    }
    return o;
}

File: codefights.com/encode.py
def encode(s):
    o = ''
    c = 0
    l = s[0]
    for x in s:
        if x == l:
            c += 1
        else:
            o += l + str(c)
            l = x
            c = 1
    return o + l + str(c)
